### PR TITLE
A getdemo name parameter, getdemo and listdemos entries in usage.cfg

### PR DESCRIPTION
**Pull request by shacknetisp**
Breaks Protocol

### CODE REVIEW

**Comment by qreeves** [comment]
Cube 2 code does not use free/strdup. Please use delete/DELETE and newstring.

I'm not sure if a vectored list like this is a good idea either, everything is rather asynchronous and you can't be guaranteed that it is responding to the correct reply. I'd say the protocol needs a slight modification to include a unique request ID when getting the demo from the server.

**Comment by shacknetisp** [comment]
I deleted the wrong branch by accident.

File: src/game/game.h [hunk 1]
```diff
@@ -4,7 +4,7 @@
 #include "engine.h"
 
 #define VERSION_GAMEID "fps"
-#define VERSION_GAME 226
```
Comment (qreeves) [inline src/game/game.h]: I'd rather pull requests left the version information alone so I can do it without getting conflicts, can I get you to remove this part please?

**Comment by qreeves** [comment]
Please rebase this branch on master so I can merge.

**Pull request merged**